RUFAS/routines/manure/manure_treatments/anaerobic_digestion_and_lagoon.py

```diff
@@ -84,13 +84,23 @@ def _daily_update_helper(self) -> ManureTreatmentDailyOutput:
 
         self._adjust_accumulated_output(anaerobic_lagoon_daily_output)
 
+        self._accumulated_output.liquid_manure_nitrogen = (
+            self._anaerobic_lagoon._accumulated_output.liquid_manure_nitrogen
+        )
+        self._accumulated_output.liquid_manure_total_ammoniacal_nitrogen = (
+            self._anaerobic_lagoon._accumulated_output.liquid_manure_total_ammoniacal_nitrogen
+        )
+        self._accumulated_output.liquid_manure_total_solids = (
+            self._anaerobic_lagoon._accumulated_output.liquid_manure_total_solids
+        )
+        self._accumulated_output.liquid_manure_total_volatile_solids = (
+            self._anaerobic_lagoon._accumulated_output.liquid_manure_total_volatile_solids
+        )
         self._accumulated_output.liquid_manure_total_degradable_volatile_solids = (
             self._anaerobic_lagoon._accumulated_output.liquid_manure_total_degradable_volatile_solids
         )
         self._accumulated_output.liquid_manure_total_non_degradable_volatile_solids = (
             self._anaerobic_lagoon._accumulated_output.liquid_manure_total_non_degradable_volatile_solids
         )
 
```

Comment:
```suggestion

anaerobic_lagoon_daily_output.biogas = self.anaerobic_digestion_daily_output.biogas
```
Reverting removal of this output to maintain backwards compatibility